### ISSUE TITLE
Inputs for BuildHelper do not work.

### ISSUE THREAD

**Issue by EtagereQC** [mokojm/Townscaper-BuildHelper#2]
**When I open the game, MelonLoader gives this error (doesn't prevent me from playing)**: 
`[ERROR] Exception in IL2CPP-to-Managed trampoline, not passing it to il2cpp: System.NullReferenceException: Object reference not set to an instance of an object
  at BuildHelper.BuildHelperMain.ResetSphere () [0x00001] in <eeb1255b537d4930a3c2114ef8b9b8fb>:0
  at BuildHelper.HelperUI.FixedHeightToggle (System.Boolean value) [0x0000d] in <eeb1255b537d4930a3c2114ef8b9b8fb>:0
  at BuildHelper.HelperUI+<>c.<Initialize>b__4_0 (System.Boolean value) [0x00001] in <eeb1255b537d4930a3c2114ef8b9b8fb>:0
  at (wrapper dynamic-method) UnhollowerRuntimeLib.DelegateSupport.(il2cpp delegate trampoline) System.Void_System.Boolean(intptr,bool,UnhollowerBaseLib.Runtime.Il2CppMethodInfo*)`
![Capture d’écran 2022-10-22 230829](https://user-images.githubusercontent.com/9853137/197371456-68ac5e35-993e-46d0-8cc8-9e6feda7599f.png)

**As soon as I place a block, with or without BuildHelper, MelonLoader gives this error (doesn't prevent from playing. However, BuildHelper still doesn't work)**: 
`[ERROR] Exception in Harmony patch of method void Placemaker.HoverData::SetHover(UnityEngine.Vector2 position, int actionId):
System.NullReferenceException: Object reference not set to an instance of an object
  at BuildHelper.BuildHelperMain.Initialize (Placemaker.HoverData thisHover) [0x000d8] in <eeb1255b537d4930a3c2114ef8b9b8fb>:0
  at BuildHelper.Harmony_Main+BuildHelper.Postfix (Placemaker.HoverData& __instance) [0x0000e] in <eeb1255b537d4930a3c2114ef8b9b8fb>:0
  at (wrapper dynamic-method) Placemaker.HoverData.DMD<Placemaker.HoverData::SetHover>(Placemaker.HoverData,UnityEngine.Vector2,int)
  at (wrapper dynamic-method) MonoMod.Utils.DynamicMethodDefinition.DMD<Placemaker.HoverData::SetHover>_il2cpp(intptr,UnityEngine.Vector2,int)`
![Capture d’écran 2022-10-22 230928](https://user-images.githubusercontent.com/9853137/197371458-a18adb25-1a52-4214-9472-33f9e1900182.png)

BuildHelper is in the "Mods" folder.

**Comment by mokojm** [comment]
Hi ! What version of the game do you use ? Did you get it on GOG ?
The game version is different from Steam one and is not compatible for now with BuildHelper.

**Comment by EtagereQC** [comment]
> Hi ! What version of the game do you use ? Did you get it on GOG ? The game version is different from Steam one and is not compatible for now with BuildHelper.

Yep, I'm using the GOG version. Well that's a damn shame! Hopefully it will be updated. Salutations du Québec  (•◡•) /

**Comment by Paldiu** [comment]
I'm currently having this same exact issue with the Steam version.

**Comment by GenericAsian526** [comment]
I am also having the same issue with the steam version.

**Comment by mokojm** [comment]
Hi everyone, 
The issue is related to the new patch version that was delivered a while ago in GOG and has been delivered in Steam recently. I'm currently working on a fix. It could be delivered by the end of the week.

**Comment by Pendzoncymisio** [comment]
Can I assume that it won't be fixed and this beautiful mod will be unusable forever?
